store/tools/ics23/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/cosmos/cosmos-sdk/store/tools/ics23
+
+go 1.17
+
+require (
+	github.com/confio/ics23/go v0.7.0
+	github.com/lazyledger/smt v0.2.1-0.20210709230900-03ea40719554
+	github.com/tendermint/iavl v0.13.2
```

Comment:
could we bump this to the latest version of iavl. https://github.com/cosmos/iavl/releases/tag/v0.17.3

Reply:
Done